extensions/wikia/Wall/WallTabsRenderer.php

```diff
@@ -45,6 +45,11 @@ public function renderUserPageContentActions( array &$contentActions ) {
 
 	public function renderWallContentActions( array &$contentActions ) {
 		$messageTitle = $this->getMessageTitle();
+
+		if ( empty( $messageTitle ) ) {
+			return;
+		}
+
 		$wallTitleText = $messageTitle->getBaseText();
 
 		$owningUser = User::newFromName( $wallTitleText, $validateUserName = false );
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "$validateUserName" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1481)